### PR TITLE
Added CN Milvus Insight docs.

### PR DESCRIPTION
**Pull request by YiyunNi**

### CODE REVIEW

**Review by shanghaikid — APPROVED** [review]

File: site/zh-CN/userGuide/tools/insight_overview.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+了解如何 [安装 Milvus Insight](insight_install.md).
+
+
+
+## 贡献代码
+
+欢迎向开源项目 Milvus Insight 贡献代码。贡献代码前，请先阅读[贡献指南](https://github.com/milvus-io/milvus-insight#-building-and-running-milvus-insight-andor-contributing-code)。
+
+请通过创建 [GitHub Issue](https://github.com/milvus-io/milvus-insight/issues/new/choose) 来提交 bug 或建议新功能。请不要重复创建相同的 issue。
```
Comment (LocoRichard) [inline site/zh-CN/userGuide/tools/insight_overview.md:50]: *报告* bug

File: site/zh-CN/userGuide/tools/insight_install.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+启动 Docker 后，打开浏览器并输入地址 `http://{ your machine IP }:8000` 以访问 Milvus Insight。
+
+![Insight_install](../../../../assets/insight_install.png)
+
+## 贡献代码
+
+欢迎向开源项目 Milvus Insight 贡献代码。贡献代码前，请先阅读[贡献指南](https://github.com/milvus-io/milvus-insight#-building-and-running-milvus-insight-andor-contributing-code)。
+
+请通过创建 [GitHub Issue](https://github.com/milvus-io/milvus-insight/issues/new/choose) 来提交 bug 或建议新功能。请不要重复创建相同的 issue。
```
Comment (LocoRichard) [inline site/zh-CN/userGuide/tools/insight_install.md]: *报告* bug

File: site/zh-CN/userGuide/tools/insight_install.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+安装 Milvus Insight 前，请先确保你已安装[单机版 Milvus](https://milvus.io/docs/install_standalone-docker.md) 或[分布式版 Milvus](https://milvus.io/docs/install_cluster-docker.md)。
+
+<div class="alert note">
+Milvus Insight 仅支持 Milvus 2.0 或以上版本。
+</div>
+
+## 启动 Milvus Insight 实例
+
+```Apache
+docker run -p 8000:3000 -e HOST_URL=http://{ your machine IP }:8000 -e MILVUS_URL={your machine IP}:19530 milvusdb/milvus-insight:latest
```
Comment (LocoRichard) [inline site/zh-CN/userGuide/tools/insight_install.md:18]: Just a proposal - in previous docs we use *<your_machine_ip>* kind of style.

**Review by LocoRichard — CHANGES_REQUESTED** [review]

File: site/zh-CN/userGuide/tools/insight_install.md (hunk 3 above)
Comment (YiyunNi) [reply site/zh-CN/userGuide/tools/insight_install.md:18]: @shanghaikid Can we change the style as proposed above?

**Pull request merged**